### ISSUE TITLE
Pandoc replaces aria-describedby in image attribute with data-aria-describedby.

### ISSUE THREAD

**Issue by BenKey** [jgm/pandoc#5642]
# Problem description

In one of my Markdown documents I attempted to set the [aria-describedby attribute](https://www.w3.org/TR/wai-aria/#aria-describedby) for an image as follows.

## Image Reference

```
::::: {.centered-element}
![ActiveX control Windows Security Warning dialog][]
:::::
```
## Image Definition

`[ActiveX control Windows Security Warning dialog]: <https://sullivanandkey.com/img/ActiveX-control-Windows-Security-Warning-dialog.png> {.centered-image aria-describedby="ActiveX-control-Windows-Security-Warning-dialog-longdesc"}
`

## Output

The correct HTML output for this is as follows.

```
<figure>
<img src="https://sullivanandkey.com/img/ActiveX-control-Windows-Security-Warning-dialog.png" alt="ActiveX control Windows Security Warning dialog" class="centered-image" aria-describedby="ActiveX-control-Windows-Security-Warning-dialog-longdesc" /><figcaption>ActiveX control Windows Security Warning dialog</figcaption>
</figure>
```

The output that was produced was as follows.

```
<figure>
<img src="https://sullivanandkey.com/img/ActiveX-control-Windows-Security-Warning-dialog.png" alt="ActiveX control Windows Security Warning dialog" class="centered-image" data-aria-describedby="ActiveX-control-Windows-Security-Warning-dialog-longdesc" /><figcaption>ActiveX control Windows Security Warning dialog</figcaption>
</figure>
```

This behavior is inconsistent with the Pandoc documentation, which says the following.

> For HTML and EPUB, all attributes except width and height (but including srcset and sizes) are passed through as is.
> The other writers ignore attributes that are not supported by their output format.

Prepending data- to an attribute is by no means passing the attribute through as is.

# System Information

**Pandoc version:** 2.4
**OS:** Ubuntu 18.04.2 LTS

**Comment by jgm** [comment]
We have a whitelist of legitimate HTML5 attributes and prepend data- to others (creating a custom attribute which is legal in HTML5).  But attributes beginning `aria-` should be whitelisted.  The documentation should also be updated.  I'll do both things now.

**Comment by BenKey** [comment]
Thanks.

**Comment by mb21** [comment]
closed by 6d30d3e0b3b4a7005018179ff1098dd16eeb2652
